packager/hls/base/master_playlist.cc

```diff
@@ -218,28 +218,34 @@ void BuildStreamInfTag(const MediaPlaylist& playlist,
                     variant.text_codecs.end());
   tag.AddQuotedString("CODECS", base::JoinString(all_codecs, ","));
 
+  const std::string I_Frame_Tag("#EXT-X-I-FRAME-STREAM-INF");
+
   uint32_t width;
   uint32_t height;
   if (playlist.GetDisplayResolution(&width, &height)) {
     tag.AddNumberPair("RESOLUTION", width, 'x', height);
 
     // Right now the frame-rate returned may not be accurate in some scenarios.
     // TODO(kqyang): Fix frame-rate computation.
-    const double frame_rate = playlist.GetFrameRate();
-    if (frame_rate > 0)
+    if (tag_name.compare(I_Frame_Tag) != 0) {
+      const double frame_rate = playlist.GetFrameRate();
+      if (frame_rate > 0)
       tag.AddFloat("FRAME-RATE", frame_rate);
+    }
 
     const std::string video_range = playlist.GetVideoRange();
     if (!video_range.empty())
       tag.AddString("VIDEO-RANGE", video_range);
   }
 
-  if (variant.audio_group_id) {
-    tag.AddQuotedString("AUDIO", *variant.audio_group_id);
-  }
+  if (tag_name.compare(I_Frame_Tag) != 0) {
```

Comment:
This change shouldn't be needed variant.audio_group_id and variant.text_group_id are null for iframe only playlist.

Reply:
Change removed